mscore/newwizard.cpp

```diff
@@ -150,6 +150,27 @@ TitleWizard::TitleWizard(QWidget* parent)
    : QWidget(parent)
       {
       setupUi(this);
+      connect(this->title, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->subtitle, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->composer, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->poet, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      connect(this->copyright, SIGNAL(cursorPositionChanged(int, int)), this, SLOT(characterAfterCursor(int, int)));
+      }
+
+void TitleWizard::characterAfterCursor(int old_pos, int new_pos)
+      {
+      QLineEdit* edit = static_cast<QLineEdit*>(QObject::sender());
+      if (edit) {
+            QChar c = edit->text().at(new_pos);
+            QString descr;
+            if (c == ' ')
+                  descr = "space";
+            else
+                  descr = QString(c);
+            edit->setAccessibleDescription(descr);
+            QAccessibleEvent event(edit, QAccessible::DescriptionChanged);
```

Comment:
[According to the Qt docs](https://doc-snapshots.qt.io/qt5-dev/qaccessible.html#Event-enum), `QAccessible::DescriptionChanged` refers to the description of the widget, not the contents of the widget. I realise that you are just using this to tell the screen reader "say something now", but ideally you should send a `QAccessible::TableSummaryChanged` event, or a `QAccessible::TextColumnChanged` event. The descriptions for these events include:

- `QAccessible::TableSummaryChanged` 
  - > "[...] The caret has moved in an editable widget. [...]"
- `QAccessible::TextColumnChanged`
  - > "A text column has been changed. [...] The selected text has changed in an editable widget. [...]"

Of course it is possible that Qt is already sending those events but the screen reader ignores them, or there is no way to say which text has changed (i.e. request a particular line or letter is spoken). If this is the case you can do it your way, but you need to write a comment to say why you didn't do it the proper way.